docs/v0.9/en/reference/sql/explain.md

```diff
@@ -35,21 +35,51 @@ The column `plan_type` indicates whether it's a`logical_plan` or `physical_plan`
-              RepartitionExec: partitioning=RoundRobinBatch(10), metrics=[fetch_time=386.585µs, repart_time=1ns, send_time=7.833µs]
-                ExecutionPlan(PlaceHolder), metrics=[]
-                
+*************************** 1. row ***************************
+stage: 0
+ node: 0
+ plan:  MergeScanExec: peers=[5471788335104(1274, 0), 5471788335105(1274, 1), ] metrics=[output_rows: 0, greptime_exec_read_cost: 0, finish_time: 1496211, ready_time: 846828, first_consume_time: 1491941, ]
+
+*************************** 2. row ***************************
+stage: 1
+ node: 0
+ plan:  CoalesceBatchesExec: target_batch_size=8192 metrics=[output_rows: 0, elapsed_compute: 4147, ]
+  FilterExec: host@2 = host1 metrics=[output_rows: 0, elapsed_compute: 32, ]
+    RepartitionExec: partitioning=RoundRobinBatch(32), input_partitions=8 metrics=[repart_time: 8, fetch_time: 230515, send_time: 256, ]
+      SeqScan: region=5471788335105(1274, 1), partition_count=0 (0 memtable ranges, 0 file ranges) metrics=[output_rows: 0, mem_used: 0, ]
+
+*************************** 3. row ***************************
+stage: 1
+ node: 1
+ plan:  CoalesceBatchesExec: target_batch_size=8192 metrics=[output_rows: 0, elapsed_compute: 3660, ]
+  FilterExec: host@2 = host1 metrics=[output_rows: 0, elapsed_compute: 32, ]
+    RepartitionExec: partitioning=RoundRobinBatch(32), input_partitions=8 metrics=[repart_time: 8, fetch_time: 113774, send_time: 256, ]
+      SeqScan: region=5471788335104(1274, 0), partition_count=0 (0 memtable ranges, 0 file ranges) metrics=[output_rows: 0, mem_used: 0, ]
+
+*************************** 4. row ***************************
+stage: NULL
+ node: NULL
+ plan: Total rows: 0
+4 rows in set (0.002 sec)                
 ```
 
-TODO: explains the output of `ANALYZE`.
+In above example, we got 4 rows of output. Except the last row gives an overall summary, each remains row represents a staged node of the execution plan. The `stage` column indicates the stage number, and the `node` column indicates the node number. The `plan` column explains the plan and related metrics in detail.
+
+### Stage and node
+
+This example is based on a distributed partition table with two regions. We mark different "stage" in a distributed execution plan with stage number, start from 0. And in each stage, we may have multiple "node" to execute the plan in parallel. The node number also starts from 0 in each stage.
+
+### Metrics
+
+Each plan node has its own metrics after the word `metrics`. The metrics may or may not include `output_rows`, `elapsed_compute`, `repart_time`, `fetch_time`, `send_time`, `mem_used` etc. They can be classified into several categories:
+- Timer
+  - `elapsed_compute`, `repart_time`, `fetch_time`, `send_time` etc.
+  - Shows the elapsed time in nanosecond. Different timer corresponding different time span.
+  - Notice that for plans that have multiple partitions, the timer is usually summed across all partitions.
```

Comment:
The explanation is not clear enough and needs to be more explicit.

Reply:
Could you please figure out which specific place is not clear?

Reply:
I'm not sure, I have two questions when I read this line:
1. what's the detailed summing logic across all partitions?
2. do all the timers use the summing logic? it doesn't seem reasonable for `fetch_time` or `send_time`

Reply:
please checkout [7bdfabe](https://github.com/GreptimeTeam/docs/pull/1114/commits/7bdfabe260168ca62094ab1fc12119b3e40ae793)

"summing" is using "add" operation to reduce many numbers into one, is there any ambiguity?